src/index.ts

```diff
@@ -94,6 +94,15 @@ export class Transport {
 		}
 	}
 
+	/**
+	 * Closes the channel.
+	 */
+	public async closeChannel() {
+		if (this.publishChannel) {
```

Comment:
What about the subscription channel?

Reply:
Added close subscriber channel when closing the connection